tailwind.config.ts

```diff
@@ -1,9 +1,25 @@
 import type { Config } from "tailwindcss";
+import defaultTheme from "tailwindcss/defaultTheme";
 
 export default {
     content: ["./{src,docs,blog}/**/*.{jsx,tsx,html}"],
     theme: {
+        screens: {
+            xs: "380px",
+            ...defaultTheme.screens,
+        },
+        fontFamily: {
+            sans: ["Jost", "sans-serif"],
+            mono: ["Roboto Mono", "monospace"],
+        },
         extend: {},
     },
+    safelist: [
+        // Classes used for code highlighting
+        "text-green-400",
+        "text-pink-500",
+        "text-violet-400",
+        "italic",
```

Comment:
Why we should describe these classes here?

Reply:
Hm, I've a comment above to explain that, guess it should be more comprehensive.

When building, Tailwind only includes classes you actually use in the output. These classes we use for code highlighting, but since code highlighting happens in runtime, Tailwind can't possibly know that we'll need those unless we explicitly list them here.